content/getting-started/multiDeviceMultiSig/_index.en.md

```diff
@@ -0,0 +1,175 @@
+    alt = "Configure collaborative wallet (Screenshot)">
+
+
+7. Be sure to check your configuration before clicking "Continue".<br/><br/>
+
+<img src = desktopCurrentConfig.png
+    alt = "your current configuration (Screenshot)">
+
+8. You can now review other parameters of your wallet configuration, then continue.<br/><br/>
+
+<img src = desktopReviewWallet.png
+    alt = "Review collaborative wallet (Screenshot)">
+
+
+9. Ensure your local key is selected (for me, that is my Desktop Key) and click "Continue".
+
+{{< notice tip >}}
+The key in green (Software) is an eligible key to sign with for your current device. Foreign keys are not eligible for signing locally and will need to be signed on the device they were imported or created on.
+{{< /notice >}}
+
+<img src = desktopAssignKeys.png
+    alt = "Assign key (Screenshot)">
+
+<img src = desktopSelectKey.png
+    alt = "Assign Key Desktop (Screenshot)">
+
+10. You should now see the original chat screen you created in step 2. This is where you can find the pending wallet configuration when you begin adding your keys on the next device.<br/><br/>
+
+<img src = desktopConfigDone.png
+    alt = "Chat screen pending keys (Screenshot)">
+
+{{< notice note >}}
+You do not need to continue to the other tabs if you were able to add all the keys to your wallet (number 9). If you want to add a key created from another device, continue to the next tab.
+{{< /notice >}}
+
+{{< /tab >}}
+
+{{< tab "Laptop" >}}
+
+{{< notice note >}}
+If you did not have all your keys available for the initial setup, you can continue from another device to continue the setup process. It is assumed that you have started to create a wallet. If you still have a pending configuration notification in your chat screen you can continue on your laptop here. If you do not, you should refer to the desktop tab to start a configuration.
+{{< /notice >}}
+
+1. Sign into your laptop and open Nunchuk. If you need help installing, refer to the install documentation found <u> [here](/getting-started/createsoftwarekey/) </u>
+<br/><br/>
+2. Navigate to the chat screen. There you should see the pending wallet notification. Click the "View config" button.<br/><br/>
+
+<img src = laptopConfigNote.png
+    alt = "pending keys config note (Screenshot)">
+
+3. Here you see the number of current keys assigned among other information about the wallet. Click the "Continue to assign keys" button.<br/><br/>
+
+<img src = laptopWalletConfig.png
+    alt = "wallet configuration (Screenshot)">
+
+4. You can now assign the key stored locally on your laptop. Notice that the other two keys are considered foreign and your local key is recognized as a "SOFTWARE" key.<br/><br/>
+
+<img src = laptopAssignKey.png
+    alt = "assign key configuration (Screenshot)">
+
+5. After assigning your key, click the "Continue" button. You should now be returned to the chat screen and see that your config notification is now only pending 1 key.<br/><br/>
```

Comment:
You should now be returned to the chat screen and see that your wallet is now only pending 1 key.